wgpu-types/src/lib.rs

```diff
@@ -941,6 +941,43 @@ pub enum VertexFormat {
     Int4 = 29,
 }
 
+impl VertexFormat {
+    pub fn size(&self) -> u64 {
+        match self {
+            VertexFormat::Uchar2 => 2,
```

Comment:
would it be better if we group them by size?
```rust
Vf::Uchar2 |
Vf::Char2 |
Vf::Uchar2Norm |
Vf::Char2Norm => 2,
```

Reply:
Good idea :)

Reply:
Addressed in https://github.com/gfx-rs/wgpu/pull/802/commits/215c9fe75050d574041ad526b4ad378cb6e379d8